ipatests/test_integration/test_trust.py

```diff
@@ -992,3 +1052,242 @@ def test_server_option_with_unreachable_ad(self):
             self.master.run_command(['rm', '-f', ad_zone_file])
             tasks.configure_dns_for_trust(self.master, self.ad)
             self.remove_trust(self.ad)
+
+
+class TestNonPosixAutoPrivateGroup(BaseTestTrust):
+    """
+    Tests for auto-private-groups option with non posix AD trust
+    """
+    topology = 'line'
+    num_ad_domains = 1
+
+    @classmethod
+    def install(cls, mh):
+        if not cls.master.transport.file_exists('/usr/bin/rpcclient'):
```

Comment:
Why is the install method overriden instead of re-using the install method from BaseTestTrust? IMO this method definition can be removed. You can move the line `cls.existing_rangename = cls.ad_domain.upper() + "_id_range"` in the BaseTestTrust install method.